src/components/themed/ModalParts.tsx

```diff
@@ -55,37 +56,17 @@ export function ModalMessage(props: {
   )
 }
 
-export function ModalCloseArrow(props: { onPress?: () => void }) {
+export function ModalFooter({ fadeOut = false, onPress }: ModalFooterProps) {
```

Comment:
Please destructure props on their own line, `const { fadeOut = false, onPress } = props`

Reply:
We need to add `ModalFooter.bottomRem = 2.5` on its own line, after the definition of this component. This will export a constant for scrolling components to use to create the proper bottom gap. See edge-react-gui if you need an exact example of this.